### ISSUE TITLE
DRLM model for ranking

### ISSUE THREAD

**Issue by ali-fani-sd** [facebookresearch/dlrm#402]
Hi,

I wanted to check if the DLRM model can be used for ranking instead of classification. I’ve updated the loss function accordingly, but the model doesn’t seem to be performing any better. I’m evaluating performance using NDCG@5, which is around 0. In contrast, XGBoost performs significantly better on the same data.

**Comment by ZiqunYu** [comment]
Has it been resolved now?
